modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -55,7 +55,8 @@ export default class AttributeTransitionManager {
 
     for (const attributeName in this.transitions) {
       const attribute = attributes[attributeName];
-      if (!attribute || !attribute.supportsTransition()) {
+      const settings = attribute && attribute.getTransitionSetting(transitions);
+      if (!attribute || !attribute.supportsTransition() || !settings) {
```

Comment:
I don't really like this - maybe there's a refactor that streamlines it a bit?

Reply:
I believe `attribute.getTransitionSetting` already returns `null` if transition is not supported, so you only need to check `settings`